docs/concepts/pdp/overview.mdx

```diff
@@ -329,6 +329,24 @@ graph LR
     style envoy fill:#4e7ef2,color:#fff;
 ```
 
+## PDP Caching Mechanism
```

Comment:
this should have some disclaimer that it has the benefit of making the evaluation faster but potentially inaccurate